go.mod

```diff
@@ -1,3 +1,5 @@
-module github.com/magiconair/properties
+module github.com/deliveryhero/golang-properties
```

Comment:
This change needs to be removed as well.

Reply:
Ooops, fixed